install/db_scripts/preferences.php

```diff
@@ -23,6 +23,7 @@
     'homepage_login'                 => 'adm_program/overview.php',
     'theme'                          => 'simple',
     'enable_password_recovery'       => '1',
+    'enable_two_factor_authentication' => '0',
```

Comment:
for newer preferences we want to use the following syntax: `two_factor_authentication_enabled`
Would be nice if you could change that.